docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -56,97 +60,129 @@ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
 sudo apt-get update
 sudo apt-get install -y docker-ce
-sudo curl -L https://github.com/docker/compose/releases/download/1.18.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+sudo curl -L https://github.com/docker/compose/releases/download/1.22.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
 ```
 
 Install node:
 
 ```sh
-curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
 sudo apt-get install -y nodejs
 ```
 
 Install Prisma:
 
 ```sh
 npm -g install prisma
-prisma local start
 ```
 
-Prisma is now installed and running. At this point you should verify that everything is running correctly.
+Prisma is now installed. Let's connect a service next.
+
+## 4. Connect the Service
 
-### Verify Prisma Installation
+We're going to setup the infrastructure needed to run the Prisma database API on the droplet.
 
-Verify that the correct docker containers are running:
+In your droplet, create a `docker-compose.yml` file
 
 ```sh
-> docker ps
-CONTAINER ID        IMAGE                                COMMAND                  CREATED             STATUS              PORTS                    NAMES
-e5d2e028eba9        prismagraphql/prisma:1.0.0   "/app/bin/single-ser…"   51 seconds ago      Up 49 seconds       0.0.0.0:4466->4466/tcp   prisma
-42d9d5acd0e4        mysql:5.7                            "docker-entrypoint.s…"   51 seconds ago      Up 49 seconds       0.0.0.0:3306->3306/tcp   prisma-db
+touch docker-compose.yml
 ```
+In this file, add the configuration below:
 
-Run `prisma cluster list` and verify that the local cluster is in the output.
-
-Finally, connect to the `/cluster` endpoint in a browser. If your droplet has the ip `37.139.15.166` open the following webpage: `http://37.139.15.166:4466/cluster`
+```yml
+version: '3'
+services:
+  prisma:
+    image: prismagraphql/prisma:1.12
+    restart: always
+    ports:
+    - "4466:4466"
+    environment:
+      PRISMA_CONFIG: |
+        port: 4466
+        managementApiSecret: my-secret
+        databases:
+          default:
+            connector: mysql
+            host: mysql
+            port: 3306
+            user: root
+            password: prisma
+            migrations: true
+  mysql:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: prisma
+    volumes:
+      - mysql:/var/lib/mysql
+volumes:
+  mysql:
+```
 
-If a GraphQL Playground shows up, your Prisma cluster is set up correctly.
+In the config where you see `managementApiSecret`, provide a secret you would like to use to authenticate requests to the service. In our example we are using `my-secret`.
```

Comment:
"In the config where you see `managementApiSecret`, provide a secret you would like to use to authenticate requests to the service. In our example we are using `my-secret`." ->

"The `managementApiSecret` is used to secure your Prisma server. It is specified in the server's Docker configuration and later used by the Prisma CLI to authenticate its requests against the server."